solutions/sequences/ContainsDuplicate.cpp

```diff
@@ -0,0 +1,23 @@
+// https://leetcode.com/problems/contains-duplicate/
+
+#include <vector>
+#include <unordered_set>
+
+class Solution {
+public:
+    bool containsDuplicate(vector<int>& nums) {
+       std::unordered_set<int> uniqueNumbers;
+
```

Comment:
Another option is to just add all the numbers to the set and then compare sizes. if the sizes are not equal - there are duplicates.  if u need to find the fist duplicate  then your approach will suit better.